scripts/perf/vm_scripts/docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+version: '2'
+networks:
+   mysql_net:
+     driver: bridge
+services:
+  cromwell:
+    image: "broadinstitute/cromwell:${CROMWELL_VERSION_TAG}"
+    volumes:
+      - ./cromwell:/conf
+    environment:
+      - CROMWELL_PROJECT
+      - CROMWELL_EXECUTION_ROOT
+      - CROMWELL_STATSD_HOST
+      - CROMWELL_STATSD_PORT
+      - CROMWELL_STATSD_PREFIX
+      - CLOUD_SQL_DB_USER
+      - CLOUD_SQL_DB_PASSWORD
+      - JAVA_OPTS=-Dconfig.file=/conf/cromwell.conf
+    command: ["server"]
+    depends_on:
+      - cloudsql-db
+    ports:
+      - "8000:8000"
+    networks:
+      mysql_net:
+    container_name: cromwell
+  cloudsql-db:
+    image: "gcr.io/cloudsql-docker/gce-proxy:1.11"
+    ports:
+      - "127.0.0.1:3306:3306"
+    container_name: cloudsql-db
+    command: ["/cloud_sql_proxy", "-instances=broad-dsde-cromwell-perf:us-central1:${CLOUD_SQL_INSTANCES}=tcp:0.0.0.0:3306"]
+    networks:
+      mysql_net:
+
+    #  proxy:
```

Comment:
Are we not adding this stuff to the VM in the end ? Should we just remove it if not ?

Reply:
yes I'll remove it

Reply:
we should probably add it for some future scenario but is not terribly interesting from the perf perspective